### PR TITLE
[DSA 5 Tsa] make talent categories collapsible

### PR DESCRIPTION
**Pull request by Stexinator**
## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [x] Does the pull request title have the sheet name(s)? Include each sheet name.
- [ ] Is this a bug fix?
- [x] Does this add functional enhancements (new features or extending existing features) ?
- [x] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

### CODE REVIEW

**Comment by Stexinator** [comment]
@Trestolt wenn du Zeit hast bitte reviewen

File: Das_Schwarze_Auge_5_Tsa_Edition/dsa5.html [hunk 1]
```diff
@@ -5486,62 +5513,62 @@ <h2 data-i18n="magie">Magie</h2>
 			<div class="magie-section zauber">
 				<span class="filter-text" data-i18n="filter">Filter:</span>
 				<span class="filter-text"></span>
-				<input name="attr_zauber_selector" type="radio" class="zauber-cb0" title="all" value="0" checked></input>
+                <input name="attr_zauber_selector" type="radio" class="zauber-cb0" title="all" value="0" checked/>
 				<span class="zauber-cb0" data-i18n="alle">Alle</span>
 				<span class="filter-text"></span>
-				<input name="attr_zauber_selector" type="radio" class="zauber-cb1" title="Zauberspruch" value="1"></input>
+                <input name="attr_zauber_selector" type="radio" class="zauber-cb1" title="Zauberspruch" value="1"/>
```
Comment (andernath) [inline Das_Schwarze_Auge_5_Tsa_Edition/dsa5.html:5519]: Same like title="sonderfertigkeit"

File: Das_Schwarze_Auge_5_Tsa_Edition/dsa5.html [hunk 2]
```diff
@@ -5797,32 +5824,36 @@ <h2 data-i18n="goetterwirken">Götterwirken</h2>
...
+                <input name="attr_liturgie_selector" type="radio" class="liturgie-cb0" title="all" value="0" checked/>
 				<span class="liturgie-cb0" data-i18n="alle">Alle</span>
 				<span class="filter-text"></span>
-				<input name="attr_liturgie_selector" type="radio" class="liturgie-cb1" title="liturgiespruch" value="1"></input>
+                <input name="attr_liturgie_selector" type="radio" class="liturgie-cb1" title="liturgiespruch"
+                       value="1"/>
 				<span class="liturgie-cb1" data-i18n="segen">Segen</span>
 				<span class="filter-text"></span>
-				<input name="attr_liturgie_selector" type="radio" class="liturgie-cb2" title="liturgiespruch" value="2"></input>
+                <input name="attr_liturgie_selector" type="radio" class="liturgie-cb2" title="liturgiespruch"
```
Comment (andernath) [inline Das_Schwarze_Auge_5_Tsa_Edition/dsa5.html:5834]: Same like title="sonderfertigkeit"

File: Das_Schwarze_Auge_5_Tsa_Edition/dsa5.html [hunk 3]
```diff
@@ -6359,41 +6390,41 @@ <h2 data-i18n="zustaende">Zustände</h2>
 			<h2 data-i18n="effekte">Effekte</h2>
 			<span class="filter-text" data-i18n="filter">Filter:</span>
 			<span class="filter-text"></span>
-			<input name="attr_effekt_selector" type="radio" class="effekt-cb0" title="all" value="0" checked></input>
+            <input name="attr_effekt_selector" type="radio" class="effekt-cb0" title="all" value="0" checked/>
 			<span class="effekt-cb0" data-i18n="alle">Alle</span>
 			<span class="filter-text"></span>
-			<input name="attr_effekt_selector" type="radio" class="effekt-cb1" title="effekt" value="1"></input>
+            <input name="attr_effekt_selector" type="radio" class="effekt-cb1" title="effekt" value="1"/>
```
Comment (andernath) [inline Das_Schwarze_Auge_5_Tsa_Edition/dsa5.html:6396]: Same like title="sonderfertigkeit"

File: Das_Schwarze_Auge_5_Tsa_Edition/dsa5.html [hunk 4]
```diff
@@ -489,49 +489,60 @@ <h3 data-i18n="sonstiges">sonstiges</h3>
...
 			<span class="filter-text" data-i18n="filter">Filter:</span>
 			<span class="filter-text"></span>
-			<input name="attr_sonderfertigkeit_selector" type="radio" class="sonderfertigkeit-cb0" title="all" value="0" checked></input>
+            <input name="attr_sonderfertigkeit_selector" type="radio" class="sonderfertigkeit-cb0" title="all" value="0"
+                   checked/>
 			<span class="sonderfertigkeit-cb0" data-i18n="alle">Alle</span>
 			<span class="filter-text"></span>
-			<input name="attr_sonderfertigkeit_selector" type="radio" class="sonderfertigkeit-cb1" title="sonderfertigkeit" value="1"></input>
+            <input name="attr_sonderfertigkeit_selector" type="radio" class="sonderfertigkeit-cb1"
+                   title="sonderfertigkeit" value="1"/>
```
[the remark this answers is not in the document]
Comment (Stexinator) [reply Das_Schwarze_Auge_5_Tsa_Edition/dsa5.html:497]: Das liegt daran, dass die Übersetzung für den Teil nicht fertig gemacht wurde. Ich nehm die Attribute raus, da es eh nicht richtig funktioniert

**Pull request merged**